contracts/test/forkMainnet/AMMWrapperWithPath/SetupTest.t.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "contracts-test/forkMainnet/AMMWrapperWithPath/Setup.t.sol";
+
+contract TestAMMWrapperWithPathSetup is TestAMMWrapperWithPath {
+    function testTokensTotalSupplyWhenSetup() public {
```

Comment:
What's the reason to check total supply of tokens in `AMMWrapper` test case? Also I think allowance checking could be removed as well. cc @pilagod @NIC619

Reply:
Since I saw that AMMWrapper has check total supply of tokens, I think AMMWrapperWithPath should also have...
Maybe we can discuss this and discuss what's the ideal relevant test granularity of separated files next Monday!

Reply:
The check was to make sure the token we are interacting with is correct in a fork mainnet test. We can remove it if we don't think it's worthy.